service/initialise.go

```diff
@@ -67,17 +73,22 @@ func (e *Init) DoGetHTTPServer(bindAddr string, router http.Handler) HTTPServer
 }
 
 // GetZebedee return Zebedee client
-func (e *ExternalServiceList) GetZebedee(cfg *config.Config) clients.ZebedeeClient {
-	zebedeeClient := e.Init.DoGetZebedeeClient(cfg)
+func (e *ExternalServiceList) GetZebedee(cfg *config.Config, hcCli *health.Client) clients.ZebedeeClient {
+	zebedeeClient := e.Init.DoGetZebedeeClient(cfg, hcCli)
 	e.ZebedeeCli = true
 	return zebedeeClient
 }
 
 // DoGetZebedeeClient gets and initialises the Zebedee Client
-func (e *Init) DoGetZebedeeClient(cfg *config.Config) clients.ZebedeeClient {
+func (e *Init) DoGetZebedeeClient(cfg *config.Config, hcCli *health.Client) clients.ZebedeeClient {
 	httpClient := dpHTTP.NewClient()
-	httpClient.SetTimeout(cfg.ZebedeeClientTimeout) // Published Index takes about 10s to return so add a bit more
-	zebedeeClient := zebedee.NewClientWithClienter(cfg.ZebedeeURL, httpClient)
+
+	// published index takes about 10s to return so add a bit more
```

Comment:
Maybe update comment:

```suggestion
	// as of 06/10/2022 published index takes about 10s to return so add a bit more, this could increase or decrease in the future
```

Reply:
Done